src/utilities/isPalindrome/isPalindrome.test.js

```diff
@@ -0,0 +1,20 @@
+import { isPalindrome } from './isPalindrome';
+
+describe('The isPalindrome function', () => {
+  describe('when provided with palindrome', () => {
+    it('should return true', () => {
+      const result = isPalindrome('Kayak');
+      expect(result).toBe(true);
+      const result2 = isPalindrome('Was it a cat I saw');
```

Comment:
`result2` is a very, very bad name. Create a separate case using `describe` for strings with spaces, and a separate case for strings with uppercase characters